shakenfist/util.py

```diff
@@ -117,6 +117,21 @@ def get_libvirt():
     return LIBVIRT
 
 
+def extract_power_state(libvirt, domain):
+    state, _ = domain.state()
+    if state == libvirt.VIR_DOMAIN_SHUTOFF:
+        return 'off'
+    elif state == libvirt.VIR_DOMAIN_CRASHED:
+        return 'crashed'
+    elif state in [libvirt.VIR_DOMAIN_PAUSED,
+                   libvirt.VIR_DOMAIN_PMSUSPENDED]:
+        return 'paused'
+    else:
```

Comment:
This else is not really necessary now it's a function. (Nor the elif.)

Reply:
Nah, I'll fix it.